core/src/main/java/nu/yona/server/subscriptions/service/UserDTO.java

```diff
@@ -30,19 +30,17 @@
 	private final String mobileNumber;
```

Comment:
I would like to rename emailAddress to emailAddressForInvitation or something like that to prevent misunderstanding.